[2023-12-10 19:32:59,368] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: mi_primer_dag_v3.segunda_tarea scheduled__2022-10-16T00:00:00+00:00 [queued]>
[2023-12-10 19:32:59,519] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: mi_primer_dag_v3.segunda_tarea scheduled__2022-10-16T00:00:00+00:00 [queued]>
[2023-12-10 19:32:59,520] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2023-12-10 19:32:59,522] {taskinstance.py:1377} INFO - Starting attempt 1 of 6
[2023-12-10 19:32:59,523] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2023-12-10 19:32:59,850] {taskinstance.py:1397} INFO - Executing <Task(BashOperator): segunda_tarea> on 2022-10-16 00:00:00+00:00
[2023-12-10 19:32:59,903] {standard_task_runner.py:52} INFO - Started process 986 to run task
[2023-12-10 19:33:00,107] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'mi_primer_dag_v3', 'segunda_tarea', 'scheduled__2022-10-16T00:00:00+00:00', '--job-id', '236', '--raw', '--subdir', 'DAGS_FOLDER/primer_dag_v3.py', '--cfg-path', '/tmp/tmpduj3okk9', '--error-file', '/tmp/tmp3s12rr1e']
[2023-12-10 19:33:00,132] {standard_task_runner.py:80} INFO - Job 236: Subtask segunda_tarea
[2023-12-10 19:33:00,798] {task_command.py:371} INFO - Running <TaskInstance: mi_primer_dag_v3.segunda_tarea scheduled__2022-10-16T00:00:00+00:00 [running]> on host 42828894a739
[2023-12-10 19:33:02,706] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=DavidBU
AIRFLOW_CTX_DAG_ID=mi_primer_dag_v3
AIRFLOW_CTX_TASK_ID=segunda_tarea
AIRFLOW_CTX_EXECUTION_DATE=2022-10-16T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-10-16T00:00:00+00:00
[2023-12-10 19:33:02,708] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2023-12-10 19:33:02,721] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'echo hola, soy la tarea 2 y sere corrida luego de la Tarea 1']
[2023-12-10 19:33:02,847] {subprocess.py:85} INFO - Output:
[2023-12-10 19:33:02,872] {subprocess.py:92} INFO - hola, soy la tarea 2 y sere corrida luego de la Tarea 1
[2023-12-10 19:33:02,873] {subprocess.py:96} INFO - Command exited with return code 0
[2023-12-10 19:33:03,297] {taskinstance.py:1420} INFO - Marking task as SUCCESS. dag_id=mi_primer_dag_v3, task_id=segunda_tarea, execution_date=20221016T000000, start_date=20231210T193259, end_date=20231210T193303
[2023-12-10 19:33:03,495] {local_task_job.py:156} INFO - Task exited with return code 0
[2023-12-10 19:33:04,000] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
